### DISCUSSION TITLE
VulkanSceneGraph-1.0.4

### DISCUSSION THREAD

**Discussion by vsg-dev** [vsg-dev/VulkanSceneGraph#757] [Announcements]
## What's Changed
* Creation of dedicated [VulkanSceneGraph website](http://www.vulkanscenegraph.org/) which provides a new home for documentation previously held within the VulkanSceneGraph repository, and adds online reference docs and links to tutorials etc.
* Replace RTX Mesh Shaders classes with cross vendor Khronos Mesh Shaders. by @robertosfield in https://github.com/vsg-dev/VulkanSceneGraph/pull/745
* Added SpvOptions::generateDebugInfo support to ShaderCompileSettings. by @robertosfield in https://github.com/vsg-dev/VulkanSceneGraph/pull/748 and @timoore in https://github.com/vsg-dev/VulkanSceneGraph/pull/749  
Tim Moore has written this [tutorial](https://timoore.github.io/2023/03/14/shader-source.html) on how to set up generation of shader debug info and use it with [RenderDoc](https://renderdoc.org/)
* Merge master into 1.0 branch in prep for 1.0.4 release by @vsg-dev in https://github.com/vsg-dev/VulkanSceneGraph/pull/752
* Add missing member functions for VertexDraw by @timoore in https://github.com/vsg-dev/VulkanSceneGraph/pull/754
* Merge fixes from master. by @vsg-dev in https://github.com/vsg-dev/VulkanSceneGraph/pull/755

**Full Changelog**: https://github.com/vsg-dev/VulkanSceneGraph/compare/VulkanSceneGraph-1.0.3...VulkanSceneGraph-1.0.4

<hr /><em>This discussion was created from the release <a href='https://github.com/vsg-dev/VulkanSceneGraph/releases/tag/VulkanSceneGraph-1.0.4'>VulkanSceneGraph-1.0.4</a>.</em>

**Comment by openscenegraph** [comment]
Introductory video into MeshShaders:

https://www.youtube.com/watch?v=g9FoZcEQlbA&t=33s
